src/shared_modules/utils/timeHelper.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Wazuh shared modules utils
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * December 28, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _TIME_HELPER_H
+#define _TIME_HELPER_H
+
+#include <string>
+#include <ctime>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+
+namespace Utils
+{
+    static std::string getTimestamp(std::time_t time)
```

Comment:
you should make an overload without argument that gets the std::time(nullptr) and calls this function so that you don't have to call it with std::time(nullptr) whenever you need the current timestamp. 
Or you can call it getCurrentTimestamp to be more explicit.

Reply:
It makes sense! I'll do that